src/main/java/org/springframework/samples/petclinic/model/Person.java

```diff
@@ -28,11 +28,11 @@
 public class Person extends BaseEntity {
 
 	@Column(name = "first_name")
-	@NotBlank
+	@NotBlank(message = "First Name cannot be blank")
```

Comment:
THe message is redundant and should be removed

Reply:
The message was requested in #1361